.github/issue-templates/release.md

```diff
@@ -0,0 +1,59 @@
+# Release {version}
+We are releasing a new version, this issue will keep track of the progress between the first release candidate (20th of each month) to the final release (27th of each month).
+
+After the 20th of each month we start the release process that ends 7 days after, during that period we enter a Feature Freeze. While in the Feature Freeze we do not merge new features or non critical or important bug fixes.  Fixes for bugs created by this release will be prioritized.
```

Comment:
After the 20th of each month we start the release process that ends 7 days after, during that period we enter a Feature Freeze. While in the Feature Freeze, we will only be merging bug fixes and not new features.